src/Resources/contao/templates/backend/be_tinyNews.html5

```diff
@@ -24,6 +33,7 @@ setTimeout(function() {
       }
       editor.on('focus', function(){ Backend.getScrollOffset(); });
     },
+    file_browser_callback_types: <?= json_encode($pickerTypes) ?>,
```

Comment:
The whole logic needs to be moved to the `DataContainer` class, so we can use `$this->pickerTypes` in the templates.